projects/go-lib/src/lib/components/go-loader/go-loader.component.scss

```diff
@@ -1,119 +1,67 @@
 @import '~@tangoe/gosheets/base/variables';
-
-.go-loader-container {
-  animation: loaderFadeIn 0.5s $global-transition-timing forwards;
-  display: inline-block;
-	height: 75px;
-	position: relative;
-	width: 75px;
-}
-
-.go-loader-container--small {
-  height: 50px;
-  width: 50px;
-}
-
-.go-loader-container--medium {
-  height: 100px;
-  width: 100px;
-}
-
-.go-loader-container--large {
-  height: 150px;
-  width: 150px;
-}
+@import '~@tangoe/gosheets/base/mixins';
 
 .go-loader {
-  transform-origin: 50% 50%;
-}
-
-.go-loader-gradient--finish {
-  stop-color: rgba(255, 255, 255, 0);
-}
-
-.go-loader-gradient--start-neutral {
-  stop-color: $ui-color-neutral;
-}
-
-.go-loader-gradient--start-positive {
-  stop-color: $ui-color-positive;
-}
+  display: inline-block;
+  height: 75px;
+  position: relative;
+  width: 75px;
 
-.go-loader-gradient--start-negative {
-  stop-color: $ui-color-negative;
+  path {
+    fill: $ui-color-neutral;
+    transition: fill .5s;
+  }
 }
 
-.go-loader--neutral {
-  fill: url(#goLoaderNeutralGradient);
+.go-loader--negative path {
+  fill: $ui-color-negative;
 }
 
-.go-loader--positive {
-  fill: url(#goLoaderPositiveGradient);
+.go-loader--neutral path {
+  fill: $ui-color-neutral;
 }
 
-.go-loader--negative {
-  fill: url(#goLoaderNegativeGradient);
+.go-loader--positive path {
+  fill: $ui-color-positive;
 }
 
-.go-loader--outer {
-	animation: rotatePathOuter 0.75s linear infinite;
+.go-loader--small {
+  height: 50px;
+  width: 50px;
 }
 
-.go-loader--middle {
-	animation: rotatePathMiddle 1s linear infinite;
-	transform: translate(10%, 10%);
+.go-loader--medium {
+  height: 100px;
```

Comment:
It looks like `go-loader` is set to dims of 75px x 75px, should we just set it to whatever the medium dims are? Thus, probably should change `go-loader` to 100px x 100px.